### PR TITLE
Thunderbird for Android 8.0b5 Release Notes

### PR DESCRIPTION
**Pull request by kewisch**

### CODE REVIEW

**Comment by kewisch** [comment]
@cketti @coreycb please review

**Comment by kewisch** [comment]
Ignore the version code, this is the testing output:

```
$ python scripts/render-notes.py -p net.thunderbird.android.beta 8.0b5 9999 github_notes -r kewisch/thunderbird-notes -b android_b5

===./app-thunderbird/src/main/res/raw/changelog_master.xml===
...
    <release version="8.0b5" versioncode="9999" date="2024-10-28">
        <change>Unread message colors are updated for better visibility.</change>
        <change>You will be asked for a financial contribution once you've used the app for a while. Help us continue active development on Thunderbird for Android!</change>
        <change>QR code import now suggests upgrading Thunderbird Desktop.</change>
        <change>Irish translation included and 100% complete</change>
        <change>Folder drawer width adjusted for tablet layouts</change>
        <change>New icon clarifies how to start a server-side email search.</change>
    </release>
...

===./app-metadata/net.thunderbird.android.beta/en-US/changelogs/9999.txt===
Thunderbird for Android version 8.0b5, based on K-9 Mail. Changes include:
- Unread message colors are updated for better visibility.
- You will be asked for a financial contribution once you've used the app for a while. Help us continue active development on Thunderbird for Android!
- QR code import now suggests upgrading Thunderbird Desktop.
- Irish translation included and 100% complete
- Folder drawer width adjusted for tablet layouts
- New icon clarifies how to start a server-side email search.

===github_notes===
New:
- The unread message colors have been updated to make it more apparent when a message is unread.
- We're now including a prompt to encourage users to make financial contributions. On beta, the timing of the prompt is lower than it will be on release, so you have a chance to test it.
- Added a thank you message to the funding experience, making it more obvious we are thankful.
- When using the QR code import, encourage users to upgrade Thunderbird Desktop where the feature has just landed.
- A big thank you to @aindriu80 who took the Irish translation to 100% very quickly. Irish is now available to all users.


Fixed:
- Folder drawer width is limited to the Material drawer width, improving tablet layout.
- Non-system languages are available again in the language switcher.
- Changed the code for the Gmail OAuth setup to make it more easily configurable.
- Fixed a visual glitch in the account images in the sidebar.
- When exporting settings, the correct brand name is used in the filename.
- Make the "Support Thunderbird" menu item more consistent across applications.
- With a new icon, it is now more apparent how to initiate a server-side email search.
- A few different developer efficiency and release automation changes.
- Lifecycle listener changes in hopes of reducing our user perceived crash rate.
- The website link has been updated to point to the new mobile landing page.
- Accessibility improvements for the save attachment button.


$ python scripts/render-notes.py -p com.fsck.k9 8.0b5 9999 github_notes -r kewisch/thunderbird-notes -b android_b5

===./app-k9mail/src/main/res/raw/changelog_master.xml===
...
    <release version="8.0b5" versioncode="9999" date="2024-10-28">
        <change>Unread message colors are updated for better visibility.</change>
        <change>QR code import now suggests upgrading Thunderbird Desktop.</change>
        <change>Irish translation included and 100% complete</change>
        <change>Folder drawer width adjusted for tablet layouts</change>
        <change>New icon clarifies how to start a server-side email search.</change>
    </release>
...

===./app-metadata/com.fsck.k9/en-US/changelogs/9999.txt===
- Unread message colors are updated for better visibility.
- QR code import now suggests upgrading Thunderbird Desktop.
- Irish translation included and 100% complete
- Folder drawer width adjusted for tablet layouts
- New icon clarifies how to start a server-side email search.

===github_notes===
New:
- The unread message colors have been updated to make it more apparent when a message is unread.
- Added a thank you message to the funding experience, making it more obvious we are thankful.
- When using the QR code import, encourage users to upgrade Thunderbird Desktop where the feature has just landed.
- A big thank you to @aindriu80 who took the Irish translation to 100% very quickly. Irish is now available to all users.


Fixed:
- Folder drawer width is limited to the Material drawer width, improving tablet layout.
- Changed the code for the Gmail OAuth setup to make it more easily configurable.
- Fixed a visual glitch in the account images in the sidebar.
- When exporting settings, the correct brand name is used in the filename.
- Make the "Support Thunderbird" menu item more consistent across applications.
- With a new icon, it is now more apparent how to initiate a server-side email search.
- A few different developer efficiency and release automation changes.
- Lifecycle listener changes in hopes of reducing our user perceived crash rate.
- The website link has been updated to point to the new mobile landing page.
- Accessibility improvements for the save attachment button.
```

File: android_beta/8.0beta.yml [hunk 1]
```diff
@@ -287,8 +292,100 @@ notes:
...
+      is now available to all users.
+    short_note: Irish translation included and 100% complete
+    tag: new
+    group: 5
+
+
+  # Fixed
+  - note: >
+      Folder drawer width is limited to the Material drawer width, improving tablet layout.
+    short_note: Folder drawer width adjusted for tablet layouts
```
Comment (cketti) [inline android_beta/8.0beta.yml:345]: This kind of implies that we do have a separate tablet layout.
Maybe "Fix folder drawer being too wide on some devices".

File: android_beta/8.0beta.yml [hunk 2]
```diff
@@ -287,8 +292,100 @@ notes:
...
     k9mail_only: true
     tag: fixed
     group: 4
+
+  ## Beta 5
+  # New
+  - note: >
+      The unread message colors have been updated to make it more apparent when a message is
+      unread.
+    short_note: Unread message colors are updated for better visibility.
```
Comment (cketti) [inline android_beta/8.0beta.yml:306]: Just describing the changes. Not suggesting wording for the changelog:
- The text color of message previews in the message list has been changed to be lighter (for both read and unread messages)
- The text color of read(!) messages in the message list has been changed to be lighter.

(That's from the perspective of the light theme. When using the dark theme: s/lighter/darker/).

File: android_beta/8.0beta.yml [hunk 3]
```diff
@@ -287,8 +292,100 @@ notes:
...
+
+  # Fixed
+  - note: >
+      Folder drawer width is limited to the Material drawer width, improving tablet layout.
+    short_note: Folder drawer width adjusted for tablet layouts
+    tag: fixed
+    group: 5
+
+  - note: >
+      Non-system languages are available again in the language switcher.
```
Comment (cketti) [inline android_beta/8.0beta.yml:350]: The languages were always present, selecting them just didn't change the app language.

File: android_beta/8.0beta.yml [hunk 4]
```diff
@@ -287,8 +292,100 @@ notes:
...
+    tag: fixed
+    thunderbird_only: true
+    group: 5
+
+  - note: >
+      Changed the code for the Gmail OAuth setup to make it more easily configurable.
```
Comment (cketti) [inline android_beta/8.0beta.yml:356]: No code was changed. We switched to new temporary client IDs to make a later migration to the (hopefully) final client IDs more seamless.

File: android_beta/8.0beta.yml (hunk 2 above)
Comment (kewisch) [reply android_beta/8.0beta.yml:306]: It seems to me what you describe can be summarized as it is now, "Unread message colors are updated for better visibility". Is anything missing?

Comment (cketti) [reply android_beta/8.0beta.yml:306]: The text color used for unread messages hasn't changed. The one for read messages has.

**Review by coreycb — APPROVED** [review]

**Pull request merged**